sources/packages/api/src/esdc-integration/msfaa-integration/msfaa-files/msfaa-file-detail.ts

```diff
@@ -42,7 +42,7 @@ export class MSFAAFileDetail implements MSFAARequestFileLine {
     record.appendDate(this.birthDate, DATE_FORMAT);
     record.appendDate(this.processDate, DATE_FORMAT);
     record.appendWithEndFiller(this.surname, 25, SPACE_FILLER);
-    record.appendWithEndFiller(this.givenName, 15, SPACE_FILLER);
+    record.appendWithEndFiller(this.givenName || "", 15, SPACE_FILLER);
```

Comment:
Please use the "??" that will ensure that "" will be used specifically for `null` or `undefined`.